src/OrchardCore.Modules/OrchardCore.ContentFields/Indexing/SQL/BooleanFieldIndexProvider.cs

```diff
@@ -77,7 +77,7 @@ public override void Describe(DescribeContext<ContentItem> context)
                                 ContentItemId = contentItem.ContentItemId,
                                 ContentItemVersionId = contentItem.ContentItemVersionId,
                                 ContentType = contentItem.ContentType,
-                                ContentPart = pair.Definition.PartDefinition.Name,
+                                ContentPart = pair.Definition.ContentTypePartDefinition.Name,
```

Comment:
This is not unique among content types, i.e. you can attach a named part with the name "Example" to more than one content type, even different named parts. I don't think this is an issue, but worth keeping in mind. Please check out if this causes any issues, including with SQL indices (it doesn't seem so; unless you created custom unique indices on just that column).